.github/workflows/security-center-snippets.yaml

```diff
@@ -0,0 +1,68 @@
+name: security-center-snippets
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'security-center/snippets/**'
+  pull_request:
+    paths:
+    - 'security-center/snippets/**'
+  pull_request_target:
+    types: [labeled]
+    paths:
+    - 'security-center/snippets/**'
+  schedule:
+  - cron:  '0 0 * * 0'
+jobs:
```

Comment:
```suggestion
env:
  GCLOUD_PROJECT=project-a
  GCLOUD_ORGANIZATION=1081635000895
jobs:
```

Reply:
If this doesn't work, the problem could be that the 
I'm hoping the [Service Account specified](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/fb09fa07199987ec0f6a201dce129e65a3c65514/.github/workflows/security-center-snippets.yaml#L31) in this YAML file has access to `project-a`.

Reply:
Also, I'm not sure we need `GCLOUD_PROJECT: project-a`.

Reply:
For any future passerby: [relevant Google Chat thread here](https://chat.google.com/room/AAAA6duiSb8/iyc_ohDP4qQ).
We copied this from https://github.com/googleapis/nodejs-security-center/pull/82/files#diff-d6eab912f8f028098a6804acd0b88600c8593ac861adbe78a6d38ff05fc645e8.